.vscode/launch.json

```diff
@@ -7,7 +7,8 @@
         {
             "name": "my_todo_app",
             "request": "launch",
-            "type": "dart"
+            "type": "dart",
+            "flutterMode": "debug"
```

Comment:
you may try to remove this line and try if debugging works. if not, retain this line.